content/en/workflows/build.md

```diff
@@ -149,6 +149,37 @@ All the variables of the Source object are visible in the Context Variables tab.
 
 {{< img src="workflows/context-variables-tab-source-object-variables.png" alt="The Source object variables in the Context Variables tab" >}}
 
+## Error handling and fallbacks
+
+In the event that a step fails, you can specify the number of times you want your workflow to retry the step and at what interval, before moving on to an optional fallback step. If no fallback step is provided, the workflow terminates after all retries have been exhausted.
+
+To configure error handling for a step:
+1. Click on the step in the workflow canvas.
+1. Click the **+** icon next to the **Error Handling & Retries** section.
+1. Adjust the **Interval** and **Max retries** values.
+1. Optionally, [add a fallback step](#add-a-fallback).
+1. Save your workflow to apply the changes.
+
+{{< img src="workflows/error-handling.png" alt="The Error Handling and Retries section" style="width:100%;" >}}
+
+### Add a fallback
+
+To handle a step failure, you can add a downstream workflow step as a fallback by selecting it from the **Fallback** drop-down menu.
+
+Alternatively, you can create a fallback step that branches from the main workflow tree:
+1. From the **Fallback** drop-down menu, select **Add a new fallback**. The workflow canvas is replaced with a fallback tree.
+1. Click the **+** icon on the fallback tree to add a step.
+1. [Add steps using the workflow builder](#build-a-workflow-with-the-workflow-builder). You can add as many steps as needed to the fallback tree.
+1. When you're done configuring your fallback steps, click **Save** to apply your changes.
+
+To get back to the main workflow canvas, click **Main** above the fallback tree. From the workflow canvas, a fallback icon appears next to steps with a fallback. Click the icon and select the fallback step to open the fallback tree. Alternatively, you can access the fallback tree by clicking **Edit Fallback Tree** in the **Error Handling & Retries** section of a step.
```

Comment:
I'm not able to find the **Edit Fallback Tree** option in the **Error Handling & Retries** section of my step configured with a fallback ([screenshot](https://a.cl.ly/L1uDK7RY)). Am I missing it somewhere?

Reply:
huh. I see it on both HQ and the demo org. 🤔 
https://a.cl.ly/04u69PdJ
I'll ask in the channel to make sure this is correct.